### ISSUE TITLE
wrappers.LdaMallet not working 

### ISSUE THREAD

**Issue by LalaNguyen** [piskvorky/gensim#562]
Hey there,
I'm trying to reproduce the tutorial https://radimrehurek.com/gensim/models/ldamallet.html without success. I invoked the LdaMallet via gensim.models.wrappers.LdaMallet

The results was nothing.
![screen shot 2015-12-14 at 7 19 47 pm](https://cloud.githubusercontent.com/assets/2168614/11780851/abc77f46-a297-11e5-8f10-931885e8dab2.png)

I also brought the converted model dumped by gensim to test directly on mallet. They also produced nothing.

Is there any chance that the process of converting gensim corpus to mallet corpus is incorrect ?

Looking forward to seeing your replies ...

My python : 3.5.1, scipy 0.16.0, gensim 0.12.3, numpy 1-10-1,.

**Comment by piskvorky** [comment]
There have been some fixes to gensim's LdaMallet wrapper in #516 recently (because the original java Mallet changed its file format).

What version of gensim are you running? What version of Mallet?

**Comment by LalaNguyen** [comment]
Hi piskvorky,
Still getting the error. Thank so much for the feedbacks, updating the 2 files (ldamodel.py, ldamallet.py) as suggested didn't get the errors away...

I'm using gensim 0.12.3 and mallet 2.0.8 :3 .

**Comment by piskvorky** [comment]
OK, thanks for reporting. Someone will have to look more closely, I'm not sure what the problem is here. 

Can you [gist](https://gist.github.com/) the full log (at DEBUG level) from the failing run?

CC @cscorley @jeiranj @tmylk in case you have time :)

**Comment by LalaNguyen** [comment]
Thank for your supports, here is the log file. I didn't see any debug info after setting the logging.level= DEBUG tho.

https://gist.github.com/LalaNguyen/4c401936f36c384091e3

**Comment by cscorley** [comment]
Hi @LalaNguyen, do you mind trying with 2.0.7? 

I'm also not sure that this error is coming from latest code on `develop`, the line shown in the error is not on 210: https://github.com/piskvorky/gensim/blob/280a488cd688ef8c58b823f4845169e925bda708/gensim/models/wrappers/ldamallet.py#L239

**Comment by jeiranj** [comment]
yes, agree with @cscorley.  I would try Mallet 2.0.7.  @LalaNguyen, could you let us know what the result is with Mallet 2.0.7?

**Comment by LalaNguyen** [comment]
Mm..  (1) replacing with Mallet 2.0.7 still ended up with this error. Btw, I'm testing this on my Window Server 2008 64x using Power shell. 
(2) In prior to the error, I have had some difficulties with installing scipy, I had to resort to conda to install it  and gensim worked but produce this error.
http://stackoverflow.com/questions/28190534/windows-scipy-install-no-lapack-blas-resources-found - scipy installation following maniac suggestions
(3) Last night, I removed scipy only and reinstalled via conda but then I realized that sub modules were not properly installed (i.e.cannot import _fblas.py lead to DLL import problem). 
From (1), (2) and (3), I suspect some missing behavior with corrupted scipy installation. Guess I will do a fresh install and give you guys more information later. 
Sorry for taking your time @piskvorky, @cscorley, @jeiranj  and thank for your supports !
